dev/checkbox_input.sh

```diff
@@ -137,6 +137,19 @@ remove_checkbox_instructions() {
   fi
 }
 
+# for vim movements
+on_checkbox_input_ascii() {
+  local key=$1
+  case $key in
+    "j" ) on_checkbox_input_down;;
+    "k" ) on_checkbox_input_up;;
+  esac
+}
+
+_null() {
```

Comment:
Could you remove this and use the `on_default` function already in `inquirer_common.sh`? Thanks!